cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -470,7 +471,7 @@ func (m *stateMachine) locateUnencryptedSave(ctx *context) (stateFunc, error) {
 		if _, ok := err.(disks.FilesystemLabelNotFoundError); ok {
 			// this is ok, ubuntu-save may not exist for
 			// non-encrypted device
-			ctx.degradedState.SaveState = "not-needed"
+			ctx.degradedState.SaveState = "not-found"
```

Comment:
the fact we didn't need to change any tests for this made me realize I forgot unit tests for the unencrypted degraded case, but after talking to @mvo5 those unit tests can wait until after this lands so we can get 2.48 rolling